src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+
+}
+
+.stars__star {
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-left: 2px;
+  margin-right: 2px;
+}
+
+.stars > :first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n) {
```

Comment:
```suggestion
.stars--0 .stars__star:nth-child(-n),
.stars--1 .stars__star:nth-child(-n+1),
...etc. {
```
<img width="1084" alt="Screenshot 2023-09-03 at 18 32 24" src="https://github.com/mate-academy/layout_stars/assets/135232001/a0747978-3ba8-4ee9-9616-c67fd745b03d">